packages/ui/src/ui/entries/main.tsx

```diff
@@ -16,7 +16,11 @@ function onAxiosError(e: any) {
 }
 
 function onError(e: any) {
-    const isAuthError = axios.isAxiosError(e) && e.response?.status === 401;
+    const isAuthError =
+        axios.isAxiosError(e) &&
+        e.response?.status === 401 &&
+        e.response.data.message !== 'Invalid password';
```

Comment:
It is bad idea to use `message` from response in such way. It is better to set some header from `server/controller/login` and check it here.